src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+
+**Syntax:**
+
+```
+rbd migration commit TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration commit target_pool1/target_image1
+```
+
+#### Option 2: Abort the Migration
+
+If necessary you can cancel the migration. This will:
+
+* Remove any cross-links between the images.
+* Delete the target image, restoring the system to its previous state in source image.
+
+**Syntax:**
+
+```
+rbd migration abort TARGET_POOL_NAME/TARGET_IMAGE_NAME
+```
+
+**Example:**
+
+```
+rbd migration abort targetpool1/targetimage1
+```
+
+The following example shows migrating data from one Ceph cluster to another Ceph
+cluster, here denoted _cluster1_ and _cluster2:_
+
+```
+[ceph: root@rbd-client /]# cat /tmp/native_spec
+{
+  "cluster_name": "c1",
+  "type": "native",
+  "pool_name": "pool1",
+  "image_name": "image1",
+  "snap_name": "snap1"
+}
+[ceph: root@rbd-client /]# rbd migration prepare --import-only --source-spec-path /tmp/native_spec c2pool1/c2image1 --cluster c2
+[ceph: root@rbd-client /]# rbd migration execute c2pool1/c2image1 --cluster c2
+Image migration: 100% complete...done.
+[ceph: root@rbd-client /]# rbd migration commit c2pool1/c2image1 --cluster c2
+Commit image migration: 100% complete...done.
+```
+
+### Supported Image Formats
+
+Live migration supports three primary formats:
+
+1. *Native format* uses Ceph's internal format for efficient migration
+
+The native format does not include the stream since it utilizes
+native Ceph operations. For example, to import from the
+image ``rbd/ns1/image1@snap1``, the source specification can be constructed as below:
+
+```
+{
+"type": "native",
+"pool_name": "rbd",
+"pool_namespace": "ns1",
+"image_name": "image1",
+"snap_name": "snap1"
+}
+```
+
+2. *QCOW format* is compatible with QEMU Copy-On-Write volume images.
+
+We can use the QCOW format to describe a QEMU copy-on-write (QCOW) block device.
+Both QCOW v1 and v2 formats are currently supported, with the exception of
+certain features including compression, encryption, backing files, and external
+data files. You can use the QCOW format with any supported stream source:
+
+```
+{
+    "type": "qcow",
+    "stream": {
+      "type": "file",
+  "file_path": "/mnt/image.qcow"
+    }
+}
+```
+
+3. *Raw format* is used for thick-provisioned block device exports.
+
+```
+{
+    "type": "raw",
+    "stream": {
+      "type": "file",
+      "file_path": "/mnt/image-head.raw"
+    },
+    "snapshots": [
+        {
+            "type": "raw",
+            "name": "snap1",
+            "stream": {
+              "type": "file",
+       "file_path": "/mnt/image-snap1.raw"
+            }
+        },
+    (optional oldest to newest ordering of snapshots)
+}
+```
+
+### Supported Streams
+
+Live migration supports multiple stream types for importing external data sources:
+
+#### File Stream (Local files)
+
+You can use a ``file`` stream to import from a locally accessible POSIX file source.
```

Comment:
These next few examples should all be in the imperative mood. This would get rid of the inconsistent pronouns ("You", "We") and it would obviate the modal verbs.